ipaserver/plugins/trust.py

```diff
@@ -436,7 +446,8 @@ def fetch_trusted_domains_over_dbus(myapi, forest_name):
         fetch_domains_method = intf.get_dbus_method(
                 'fetch_domains',
                 dbus_interface=DBUS_IFACE_TRUST)
-        (_ret, _stdout, _stderr) = fetch_domains_method(forest_name)
+        (_ret, _stdout, _stderr) = fetch_domains_method(
+            [forest_name]+method_options)
```

Comment:
This is going to fail with ``E226 missing whitespace around arithmetic operator``.

Reply:
Fixed. ;)